Makefile.am

```
@@ -46,7 +46,7 @@ uninstall_pattern = \
 
 install-data-local:
 	@$(call install_pattern,$(images_instdir),$(images_srcdir)/*)
-	cp $(DESTDIR)$(images_instdir)/bg_global_02.jpg $(DESTDIR)$(pkgdatadir)/desktop-background-C.jpg
+	ln -fs $(images_instdir)/bg_global_10.jpg $(DESTDIR)$(pkgdatadir)/desktop-background-C.jpg
```

I checked the Git history and there actually is a good reason why this is a copy, not a symlink, though it was buried in a code review comment.

This file ends up in a different package, `eos-default-background`, and the `eosinstaller` ostree only includes `eos-default-background`, not `eos-media`, which keeps the image size down. See db54b82c5aa2d398f19d6f217e32a1d3b3470ade, https://github.com/endlessm/eos-theme/pull/234#discussion_r63256233 and https://phabricator.endlessm.com/T10782

Please change this to something like:

```suggestion
	# This file is shipped in a separate package, eos-default-background, so cannot be a symlink
	cp $(DESTDIR)$(images_instdir)/bg_global_10.jpg $(DESTDIR)$(pkgdatadir)/desktop-background-C.jpg
```

---

Thanks, for digging. Changed this.